includes/class-amp-post-type-support.php

```diff
@@ -124,11 +124,12 @@ public static function get_support_errors( $post ) {
 			}
 		} else {
 			/*
-			 * Disabled by default for custom page templates, page on front and page for posts, unless 'amp' theme
-			 * support is present (in which case AMP_Theme_Support::get_template_availability() determines availability).
+			 * Disabled by default for custom page templates, page on front and page for posts, unless not using a Reader theme.
+			 * If the active theme is not being used for AMP, then we can't make assumptions about these about whether the
+			 * default Reader theme template will work properly for the post.
 			 */
 			$enabled = (
-				current_theme_supports( AMP_Theme_Support::SLUG )
+				( AMP_Theme_Support::READER_MODE_SLUG !== AMP_Options_Manager::get_option( Option::THEME_SUPPORT ) )
```

Comment:
Actually no, as this _should_ apply when a reader Reader theme is selected as well. Since Reader mode in general means a different template is used for AMP vs non-AMP, we need to default AMP to be off in both legacy Reader mode and Reader mode with a theme since the same template is not going to be present.

Reply:
I improved the comment before explaining why this applies to legacy and non-legacy reader mode in e5ced2b8cda7a44944a97dbf58aebf427d099046.

Reply:
Actually, you may be right… I didn't account for the fact in the new Reader mode the user will have the ability to enable "serve all templates as AMP". They are also able to turn off AMP for specific templates. So this isn't going to be necessary for the new Reader mode. It only applies to legacy Reader mode.

Reply:
Done in 80da32a67